<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>34</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>342</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="x_motor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="y_motor">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="z_motor">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="location">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>location</sender>
   <signal>toggled(bool)</signal>
   <receiver>z_motor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>location</sender>
   <signal>toggled(bool)</signal>
   <receiver>y_motor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>location</sender>
   <signal>toggled(bool)</signal>
   <receiver>x_motor</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x_motor</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>motor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_motor</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>motor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>z_motor</sender>
   <signal>valueChanged(double)</signal>
   <receiver>Form</receiver>
   <slot>motor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>location()</slot>
  <slot>motor()</slot>
 </slots>
</ui>
